### PR TITLE
feat: Error reported when the maximum value is less than the minimum value

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #966 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/renderless/src/numeric/vue.ts [hunk 1]
```diff
@@ -139,16 +139,24 @@ const initWatch = ({
   state,
   watch,
   props,
-  api
+  api,
 }: Pick<INumericRenderlessParams, 'state' | 'watch' | 'props' | 'api'>): void => {
+  
+  watch(() => [props.max, props.min], ([curMax,curMin]) => {
+    if (curMax = curMin){
+      console.log(curMax, curMin);
```
Comment (zzcr) [inline packages/renderless/src/numeric/vue.ts]: 源码里面需要删除调试代码

Comment (GaoNeng-wWw) [reply packages/renderless/src/numeric/vue.ts]: 已修复

File: packages/renderless/src/numeric/vue.ts [hunk 2]
```diff
@@ -139,16 +139,23 @@ const initWatch = ({
...
   props,
-  api
+  api,
 }: Pick<INumericRenderlessParams, 'state' | 'watch' | 'props' | 'api'>): void => {
+  
+  watch(() => [props.max, props.min], ([curMax,curMin]) => {
+    if (curMax = curMin){
+      throw new Error('[Numeric]: The maximum value should not be less than to the minimum value');
+    }
+  }, {immediate: true})
```
Comment (kagol) [inline packages/renderless/src/numeric/vue.ts]: @GaoNeng-wWw 
主要有以下问题：
1. 语句末尾不应该加分号`;`
2. 多处缺失必要的空格
3. `=` 应该是 `<`，这里估计是手误吧

```suggestion
  watch(() => [props.max, props.min], ([curMax, curMin]) => {
    if (curMax < curMin) {
      throw new Error('[Numeric]: The maximum value should not be less than to the minimum value')
    }
  }, { immediate: true })
```

File: packages/renderless/src/numeric/vue.ts [hunk 3]
```diff
@@ -139,16 +139,23 @@ const initWatch = ({
...
+  
   watch(() => props.modelValue, api.watchValue, { immediate: true })
 
   watch(() => state.isDisplayOnly, api.dispatchDisplayedValue)
 }
 
 export const renderless = (
   props: INumericProps,
-  { computed, onMounted, onUpdated, onUnmounted, reactive, watch, inject }: ISharedRenderlessParamHooks,
+  { computed, onMounted, onUpdated, onUnmounted, reactive, watch, inject}: ISharedRenderlessParamHooks,
```
Comment (kagol) [inline packages/renderless/src/numeric/vue.ts]: 这里也是存在空格问题

```suggestion
  { computed, onMounted, onUpdated, onUnmounted, reactive, watch, inject }: ISharedRenderlessParamHooks,
```

Comment (GaoNeng-wWw) [reply packages/renderless/src/numeric/vue.ts]: Fixed

File: packages/renderless/src/numeric/vue.ts (hunk 2 above)
Comment (GaoNeng-wWw) [reply packages/renderless/src/numeric/vue.ts]: Fixed

**Review by zzcr — APPROVED** [review]

**Pull request merged**